src/api/app/views/webui2/webui/project/_project_packages.html.haml

```diff
@@ -1,18 +1,12 @@
 .card-body
```

Comment:
This partial is called for both inherited packages (`@ipackages`) and packages. You can not remove things from here that easily. Also the variable `package` given as parameter when calling this partial is not used anymore.

Reply:
Oh, right. We forgot to split this up.

Reply:
you are right, there are two partials. I checked it wrong. Sorry.

But still you need to remove the `package` variable from:
` = render partial: 'project_packages', locals: { project: @project, packages: @packages }`

Reply:
Right now this partial still has a dependency on `@packages`. One could argue that we could, or even should, always show the package table. But I don't think we need to block this PR for that.

Reply:
ok